tiny_dnn/activations/elu_layer.h

```diff
@@ -0,0 +1,36 @@
+/*
+    Copyright (c) 2017, Taiga Nomi, Karan Desai
+    All rights reserved.
+
+    Use of this source code is governed by a BSD-style license that can be found
+    in the LICENSE file.
+*/
+#pragma once
+#include "tiny_dnn/activations/activation_layer.h"
+#include "tiny_dnn/layers/layer.h"
+
+namespace tiny_dnn {
+
+class elu_layer : public activation_layer {
+ public:
+  using activation_layer::activation_layer;
+
+  std::string layer_type() const override { return "elu-activation"; }
```

Comment:
I think that isocpp guidelines has some good performance principles. One of this is "Don't optimize not performance critical parts". I also think that here we could be involved in RVO or not?